_layouts/default.html

```diff
@@ -26,13 +26,16 @@
   {% endif %}
   <link rel="stylesheet" href="http://okfnlabs.org/githubActivity.js/githubActivity.css" />
   <link rel="stylesheet" href="/css/chosen.min.css" />
+  {% if page.bodyclass == "members" %}
```

Comment:
My intention was to keep the css and js in their respective sections, but I think you're right that it is better to pack them together.  My other thought was that since other pages down the line will want to include leaflet, it would be better to keep the deps in the default layout and only load if a given page has a "hasmap" attribute.  I updated the PR accordingly.